recipes-ni/initscripts-nilrt/files/lvrt-cgroup

```diff
@@ -0,0 +1,17 @@
+get_lvrt_cgroup_version()
+{
+	local lvrt_installed=$(opkg list-installed ni-labview-realtime)
```

Comment:
I think I agree with @amstewart here. Unfortunately there is no way to "interrogate the cgroup subsystem" because these are the scripts that capture cgroups for LVRT (and we're trying to skip them if LVRT is not installed)

Also, I've tried it on a 9030 and 9034 and what I'm seeing is closer to 100ms:
```
admin@NI-cRIO-9030-XXXXXXXX:~# time lvrt_installed=$(opkg list-installed ni-labview-realtime)

real    0m0.109s
user    0m0.109s
sys     0m0.000s
```

We will get to revisit this script when LVRT switches to cgroup-v2 so my proposal for now is to stick with the current approach.